client/main.lua

```diff
@@ -579,6 +582,13 @@ AddEventHandler('QBCore:Client:OnPlayerLoaded', function()
     init()
 end)
 
+AddEventHandler('QBCore:Client:OnPlayerUnload', function()
+    for _, v in pairs(shopVehs) do
+        DeleteEntity(v)
+    end
```

Comment:
```suggestion
    for i = 1, #shopVehs do
        DeleteEntity(shopVehs[i])
    end
```
https://manason.github.io/effective-fivem-lua/tables/#use-numeric-for-loops-when-iterating-over-an-array